cumulusci/tasks/apex/batch.py

```diff
@@ -33,9 +32,28 @@ def _run_task(self):
 
         self.logger.info("Job is complete.")
 
+        vals = {
+            key: value
+            for key, value in self.batch.items()
+            if key
+            in [
+                "Id",
+                "Status",
+                "ExtendedStatus",
+                "NumberOfErrors",
+                "JobItemsProcessed",
+                "TotalJobItems",
+            ]
+        }
         if not self.success:
-            self.logger.info("There were some batch failures.")
-            raise SalesforceException(self.batch["ExtendedStatus"])
+            self.logger.info("There have been some batch failures.")
+            self.logger.info("Error values:")
+            self.logger.info(repr(vals))
+            raise SalesforceException("There were import errors", vals)
```

Comment:
This is going to print something like:

    cumulusci.core.exceptions.SalesforceException: ('There were import errors', "{'Id': 'asdf', etc... }")

Did you mean to format this somehow?

Reply:
I didn't feel strongly about this, but PEP 352 advocates for a single string argument, so I've done that in e2368baa9e03caeb68b394c971de85d4836d78a7